docs/BR.md

```diff
@@ -1008,6 +1043,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
 
 CAs MUST document potential issuances that were prevented by a CAA record in sufficient detail to provide feedback to the CAB Forum on the circumstances, and SHOULD dispatch reports of such issuance requests to the contact(s) stipulated in the CAA iodef record(s), if present. CAs are not expected to support URL schemes in the iodef record other than mailto: or https:.
 
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the determinations (i.e., domain validation pass/fail, CAA permission/prohibition) made by the Primary Network Perspective from multiple additional Network Perspectives before Certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+The response returned from each corroborating Network Perspective MUST allow the CA to verify:
```

Comment:
I do not think this language should be interpreted that strictly. While this type of behavior (where the remote perspective must prove knowledge of the validation token's value without communication from the primary perspective) potentially offers some security against compromised or misbehaving remote perspectives, I do not think it adds much substantial security, and I feel it adds some unneeded complexity.

On the security side, I largely see attacks where the remote perspectives misbehave (i.e., due to compromise or misconfiguration) outside of the threat model used in this ballot draft. There are currently no other ways for the primary perspective to verify the behavior of the remote perspectives (although the logic running at the primary perspective must enforce that responses from remote perspectives cannot turn a do-not-issue decision into an issue decision).

Furthermore, I do not think not communicating the token to remote perspectives provides a significant security improvement. The adversary in the threat model of this ballot is already aware of the required token that needs to be seen by the remote perspectives. Specifically, if an adversary compromised a remote perspective R and the primary perspective asked "what token did you see for this domain?," it would be trivial for the adversary to craft a response from R claiming it saw the proper token given that this is the very token the adversary was supposed to upload to the victim's webpage. Given the value of the token is already known by the adversary, I don't think there is much security improvement between asking remote perspectives "what token did you see for this domain?" vs "did you see token %s for this domain?" While I do think checking the validity of the token by the primary provides a sanity check that remote perspectives are doing their job, I do not think this check is robust in the case of a compromised remote perspective. @orangepizza you also have a good point about there not being an elegant parallel in the CAA space.

Maybe removing "allow the CA" and "The response returned" could make this language more general:

```suggestion
Each corroborating Network Perspective MUST verify:
```

Reply:
If I recall correctly, the intent of the existing language was in response to an earlier comment.

The basis for that comment was such that the perspectives should not be obligated to determine whether or not they corroborate the primary (as this would obligate the CA to share the expected token with the perspectives), but instead, perspectives should only be responsible for collecting information and sharing it with the CA. Ultimately, it should be the CA's responsibility to determine whether a perspective corroborates the primary validation. 

I think the text should accommodate either implementation approach:
1. The CA asks whether the perspective saw $challenge and the perspective responds accordingly ("Did you see '1235ABC1235'"? ---> "Yes.")
2. The CA asks the perspective to return the results of the lookup ("What did you see?"  ---> "I saw '12345ABC12345'")

@orangepizza / @birgelee - I'll see if I can improve this language further given the above discussion and the original comment.

Reply:
`The response returned from each corroborating Network Perspective MUST allow the CA to verify, assume it replied truthfully`? if CA can trust NP's statement this problem doesn't happen. sentence feels weird though

Reply:
Throwing something to wall if it sticks: (not a native speaker)
`Either by perspective itself or CA, each corroborating Network Perspective's observation MUST verify:`

Reply:
I agree with @ryancdickson 's point that the text should accommodate either implementation. Just curious, if we changed it to simply:
`Each corroborating Network Perspective's observation MUST verify:`
would that clearly permit both interpretations?

Also just to reiterate, in the threat model for MPIC, the adversary is the one requesting a certificate for a domain it does not control and the challenge token is known to the adversary because it is the very token the CA sent to the adversary which was supposed to be uploaded to the domain. Thus, I do not think there is a major security advantage in not sharing tokens with remote perspectives and both of the implementations Ryan Dickson mentioned are good.

Reply:
@orangepizza / @birgelee --- thanks for the suggestions. 

What about:

"**_Each corroborating Network Perspective's response MUST provide the CA with the necessary information to allow it to affirmatively assess:_** 

- a. the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5; and
- b. the CA's authority to issue to the requested domain(s), as specified in Section 3.2.2.8."

Reply:
Mocked up in https://github.com/ryancdickson/staging/commit/278f3b84d991cb145f926f7caef5c211e05cf9a9.

Reply:
To me this language looks good.